ui/src/containers/Authentication/Authentication.test.tsx

```diff
@@ -30,7 +30,8 @@ describe('Authentication', () => {
   });
 
   it('it can test github login', () => {
-    const windowOpenSpy = spyOn(window, 'open');
+    window.open = jest.fn();
```

Comment:
Is this addition required as we are updating the packages ??

Reply:
yeah , test was failing because of react-script@4.0.1 have a changeLog in jest